homeassistant/components/immich/strings.json

```diff
@@ -0,0 +1,72 @@
+{
+  "common": {
+    "data_desc_url": "The URL of your immich instance. (_e.g. `https://immich.example.com`_)",
```

Comment:
Either remove the example or use placeholders

Reply:
you mean to avoid accidental translations of it?

Reply:
Yes